tests/network/probes.go

```diff
@@ -84,21 +83,19 @@ var _ = SIGDescribe("[ref_id:1182]Probes", func() {
 				By("Specifying a VMI with a readiness probe")
 				vmi = createReadyCirrosVMIWithReadinessProbe(virtClient, readinessProbe)
 
+				assertPodNotReady(virtClient, vmi)
+
+				// pod is not ready until our probe contacts the server
 				By("Starting the server inside the VMI")
 				serverStarter(vmi, readinessProbe, 1500)
 			}
 
-			// pod is not ready until our probe contacts the server
-			assertPodNotReady(virtClient, vmi)
```

Comment:
i am little confused,
IPv6 as well line 81 creates createReadyCirrosVMIWithReadinessProbe
which won't be ready until the server is started ?

if so, then u need to start the server on ipv6 as well don't you?
if the above is wrong, then maybe there will be still a flake on ipv4 even its moved to line 86,
because the pod might be ready without waiting for anything?